package.json

```diff
@@ -2,32 +2,89 @@
   "name": "serve",
   "version": "13.0.4",
   "description": "Static file serving and directory listing",
-  "scripts": {
-    "test": ""
-  },
-  "files": [
-    "bin"
-  ],
-  "repository": "vercel/serve",
-  "bin": {
-    "serve": "./bin/serve.js"
-  },
   "keywords": [
     "vercel",
     "serve",
     "micro",
     "http-server"
   ],
+  "repository": "vercel/serve",
   "license": "MIT",
+  "type": "module",
```

Comment:
Are you sure we want to use ESM? Perhaps this needs an `engines` field so the consumer knows which versions of Node.js are supported.

Reply:
Yes, we will need to use ESM as a few modules like `chalk` are now ESM only. Also, it allows us to use top level `await`.

Reply:
Maybe we could target Node 14 instead of Node 18, so we change `target` in tsconfig.json to es2019.

Reply:
For `chalk`, do we have to upgrade it to the ESM only version? Can stay with the current used version so that it doesn't cause break for old node versions?

Reply:
Old Node versions that don't support ESM are no longer even in maintainance mode (i.e., Node 12 and before). If someone wants to use `serve` with those versions, they could use older versions of `serve`. With ESM, we can target Node 14 onwards and use quite a few more features and libraries.

It's not just `chalk` - `clipboardy` and `is-port-reachable` are also ES modules. We would need to either downgrade, replace or write those modules ourselves if we want to use CJS - introducing more potential bugs or security issues.

Reply:
Changed the `target` field in `tsconfig.json` to ES2020 - which means the compiled code supports Node 14 onwards.

Reply:
I'm okay with Node 14+ support 👍